robot/web/web_c.go

```diff
@@ -57,10 +57,21 @@ type webService struct {
 	modWorkers   sync.WaitGroup
 }
 
+type stats struct {
+	RPCServer any
+}
+
 func (svc *webService) Stats() any {
-	return struct {
-		RPCServer any
-	}{svc.rpcServer.Stats()}
+	if haveLock := svc.mu.TryLock(); !haveLock {
```

Comment:
Hmm `TryLock`. You avoiding a long lock here when the web service is reconfiguring? Or worried/aware of some kind of deadlock?

Reply:
You're right, this needs to be justified in words

Reply:
Oops, also this PR is wrong -- that whole changing the schema thing.